WORKSPACE

```diff
@@ -159,12 +170,29 @@ cc_library(
     name = "zlib",
     linkstatic = 1,
     srcs = [":lib"],
-    testonly = 1,
 )
 """,
     repository = "@nixpkgs",
 )
 
+nixpkgs_package(
+    name = "c2hs",
+    attribute_path = "haskellPackages.c2hs",
```

Comment:
Why pull these from nixpkgs rather than stackage?

Reply:
Because we don't yet have `haskell_cabal_binary`.